src/exchange/adl_handler.cairo

```diff
@@ -145,9 +163,17 @@ mod AdlHandler {
             swap_handler_address,
             referral_storage_address
         );
+        self.data_store.write(IDataStoreSafeDispatcher { contract_address: data_store_address });
+        self
+            .event_emitter
+            .write(IEventEmitterSafeDispatcher { contract_address: event_emitter_address });
+        self.oracle.write(IOracleSafeDispatcher { contract_address: oracle_address });
+        self.chain.write(IChainSafeDispatcher { contract_address: chain_address });
```

Comment:
Why did you add chain here ?

Reply:
Removed leftover code that originated from a previous closed pull request.